src/neuroconv/tools/nwb_helpers/_configuration_models/_base_backend.py

```diff
@@ -13,9 +14,11 @@
 class BackendConfiguration(BaseModel):
     """A model for matching collections of DatasetConfigurations to a specific backend."""
 
-    backend: ClassVar[Literal["hdf5", "zarr"]] = Field(
-        description="The name of the backend used to configure the NWBFile."
-    )
+    backend: ClassVar[Literal["hdf5", "zarr"]]
+    data_io_class: ClassVar[Type[DataIO]]
+
+    model_config = ConfigDict(validate_assignment=True)  # Re-validate model on mutation
+
     data_io_class: Type[DataIO] = Field(description="The DataIO class that is specific to this backend.")
```

Comment:
Isn't `data_io_class:` repeated here (lines 22 and 18)?

Reply:
Whoops, good catch